src/jshint.js

```diff
@@ -2218,6 +2217,19 @@ var JSHINT = (function() {
     return this;
   }));
   prefix("new", function() {
+    var mp = metaProperty("target");
```

Comment:
> Are there likely to be other meta properties?

There is a stage 2 proposal - [`function.sent`](https://github.com/allenwb/ESideas/blob/master/Generator%20metaproperty.md)